src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -259,19 +260,31 @@ public static void AddScale(float a, ReadOnlySpan<float> src, ReadOnlySpan<int>
 
             if (Avx.IsSupported)
             {
-                AvxIntrinsics.AddScaleSU(a, src, indices, dst, count);
+                if (VectorizationMakesSense(count))
```

Comment:
We use a tertiary operator below, but we don't here. Is there are reason for the inconsistency?